plugins/destination_testing.go

```diff
@@ -22,6 +22,7 @@ type DestinationTestSuiteTests struct {
 	SkipOverwrite   bool
 	SkipDeleteStale bool
 	SkipAppend      bool
+	SkipAppendTwice bool
```

Comment:
Could we add a comment to explain what this option does? The others are fairly self-explanatory, but I'm not sure about this one

Reply:
Oh okay, reading the code, maybe it could be called `SkipSecondAppend`?

Reply:
Also, I'm curious why this is useful as a test case 🤔

Reply:
It is useful in cases like cloud storage where you can't append to an existing object the moment you closed the file as it's immutable while locally you can and of-course for PostgreSQL it's all valid.